dist/powerbi.js

```diff
@@ -65,19 +65,13 @@
 	var util_1 = __webpack_require__(4);
 	var PowerBi = (function () {
 	    function PowerBi(config) {
-	        var _this = this;
 	        if (config === void 0) { config = {}; }
-	        /**
-	         * Deprecated alias for embed.
-	         * This performed the same function as embed. Embed is more semantic to the operation performed so we conslidated to a single method.
-	         */
-	        this.get = this.embed;
 	        this.embeds = [];
 	        window.addEventListener('message', this.onReceiveMessage.bind(this), false);
 	        // TODO: Change when Object.assign is available.
 	        this.config = util_1.default.assign({}, PowerBi.defaultConfig, config);
 	        if (this.config.autoEmbedOnContentLoaded) {
-	            window.addEventListener('DOMContentLoaded', function (event) { return _this.init(document.body); }, false);
+	            this.enableAutoEmbed();
```

Comment:
Let's confirm that when the JS sdk is used via server side C# SDK that it configures the enableAutoEmbed correctly.